application/asset/js/global.js

```diff
@@ -413,12 +413,20 @@ var Omeka = {
             $(selectedActionClass).addClass('active');
         });
 
+        // Set up multi-value templates
+
+        $('.multi-value.field').each(function() {
+            var field = $(this);
+            var value = field.find('.value').first().clone();
+            field.attr('data-field-template', value.wrap('<div></div>').parent().html());
```

Comment:
You can set with `.data` just like you get with it, no need to do an end-around through `.attr`